--- mysql-connector-python-8.0.33-src/cpydist/utils.py~	2023-03-30 12:58:04.000000000 +0200
+++ mysql-connector-python-8.0.33-src/cpydist/utils.py	2024-03-21 13:22:42.266660572 +0100
@@ -200,7 +200,7 @@ def mysql_c_api_info(mysql_config):
         return _mysql_c_api_info_win(mysql_config)
 
     if os.path.isdir(mysql_config):
-        mysql_config = os.path.join(mysql_config, "bin", "mysql_config")
+        mysql_config = os.path.join(mysql_config, "bin", "mysql_config8.0")
 
     LOGGER.info("Getting MySQL information from %s", mysql_config)
 
